cmd/node/config/gasSchedule.toml

```diff
@@ -76,39 +76,62 @@
     GetBlockCoinbase    = 100
     Create              = 320000
     GetBlockDifficulty  = 100
-    ExternalCodeCopy    = 1000
-    GetExternalCodeSize = 500
+    ExternalCodeCopy    = 3000
+    GetExternalCodeSize = 2500
     GetGasLeft          = 100
     GetBlockGasLimit    = 100
     GetTxGasPrice       = 100
     Log                 = 3750
-    GetBlockNumber      = 100
+    GetBlockNumber      = 1000
     GetTxOrigin         = 100
     Finish              = 1
     Revert              = 1
     GetReturnDataSize   = 200
     ReturnDataCopy      = 500
     SelfDestruct        = 500000
-    GetBlockTimeStamp   = 100
+    GetBlockTimeStamp   = 1000
 
 [BigIntAPICost]
     BigIntNew                = 100
     BigIntByteLength         = 100
+    BigIntUnsignedByteLength = 100
+    BigIntSignedByteLength   = 100
     BigIntGetBytes           = 100
+    BigIntGetUnsignedBytes   = 100
+    BigIntGetSignedBytes     = 100
     BigIntSetBytes           = 100
+    BigIntSetUnsignedBytes   = 100
+    BigIntSetSignedBytes     = 100
     BigIntIsInt64            = 100
     BigIntGetInt64           = 100
     BigIntSetInt64           = 100
     BigIntAdd                = 100
     BigIntSub                = 100
     BigIntMul                = 600
+    BigIntTDiv               = 100
+    BigIntTMod               = 100
+    BigIntEDiv               = 100
+    BigIntEMod               = 100
+    BigIntAbs                = 100
+    BigIntNeg                = 100
+    BigIntSign               = 100
     BigIntCmp                = 100
+    BigIntNot                = 100
+    BigIntAnd                = 100
+    BigIntOr                 = 100
+    BigIntXor                = 100
+    BigIntShr                = 100
+    BigIntShl                = 100
     BigIntFinish             = 100
```

Comment:
Does this still exist? @andrei-marinica

Reply:
Removed, fixed (there is no `bigIntFinish` anymore).